components/ai_chat/core/browser/utils.h

```diff
@@ -48,6 +48,8 @@ EngineConsumer::GenerationDataCallback BindParseRewriteReceivedData(
 // 1024x768
 SkBitmap ScaleDownBitmap(const SkBitmap& bitmap);
 
+GURL GetEndpointUrl(bool premium, const std::string& path);
```

Comment:
The generic name was ok when it was `ConversationAPIClient::GetEndpointURL` but now this is just `ai_chat::GetEndpointURL`, not sure what's best here but just a nit for now

Reply:
Actually I'm confused why this was removed since I don't see anywhere else calling it